src/client/containers/BranchSelect/index.js

```diff
@@ -12,14 +12,13 @@ type SelectItemType = {
 }
 
 type BranchProps = {
-  repoId: number,
+  repoName: string,
```

Comment:
Because it appears very useless and not effective because we are doing big amount of search for a repository `name` by `id` in state, when instead we can do fetches with just a `name`. So denormalization of repositories by `fullName` will come afterwards.